guide/advice_for_new_collaborators.rst

```diff
@@ -0,0 +1,326 @@
+The fastest way to get started is to meet, either virtually
+or in person, to discussr what is involved
+in supporting a different model.
+
+If part of the team is not yet familiar with Data Assimilation (DA)
+we recommend they review the introductory material <here> and
+also look at either DartLab or the DART tutorial.
+
+
+Goals of Using DART
+-------------------
+
+DART is the Data Assimilation Research Testbed.  It is a collection of 
+tools and routines and scripts that allow users to built custom solutions
+and explore a variety of DA related efforts.  It is not a turnkey system;
+it must be built before use and is often customized based on needs and goals.
+
+DART is often used for the following types of projects:
+
+- Learning about Data Assimilation (DA)
+- Using DART with an existing model and supported observations
+- **Adding a DART interface to a new model**
+- Using new observations with DART in an existing model
+- Using both a new model and new observations with DART
+- Using DART to teach DA
+
+Everything on this "possible goals" list except adding support
+for a new model can generally be done by a single user with minimal 
+help from the DART team. Therefore the following discussion will
+focus only on adding a new model to DART.
+
+
+Should I consider using DART?
+-----------------------------
+
+DART is an ensemble-based DA system.  It makes multiple runs of a model
+with slightly different inputs and uses the statistical distribution of 
+the results to decide how to adjust the model state to be more consistent 
+with the observations.
+
+The advantage of ensemble systems is that no changes to the model
+itself are required.  The disadvantage is that multiple runs of the
+model are needed and this can be computationally expensive.
+
+Simple models can be added to DART with a single person effort, but
+larger, more complex models can require multiple person-months with
+support from the DART team to add the interfaces and scripts needed 
+to perform a large-scale DA experiment.
+
+The DART code is in Fortran.  The supporting scripts and tools are
+a mix of shell and python.  The model can be written in any language;
+it will only be run and the input and output files will be used by DART.
+
+
+Things to discuss before beginning
+----------------------------------
+
+Is your model appropriate for any kind of DA?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If the model is deterministic instead of chaotic, you don't need DA.
+You run the model, you look at the difference between the prediction and
+the observations, and you invert the equations inside the model to compute
+what different inputs would have produced outputs closer to the observations.
+
+Chaotic models do not have a simple relationship between inputs and
+outputs.  There are internal feedbacks and non-linear behaviors that make
+it difficult to adjust the inputs to make the outputs better match the
+observations.  
+
```

Comment:
```suggestion
Can the model be stopped and restarted and get an answer very similar to (ideally identical to) the same answer it would have gotten if it had not stopped?
```

Reply:
@johnsonb-ucar  ... incorporate the gist.

Reply:
done